ani-cli

```diff
@@ -398,6 +405,7 @@ episode_selection () {
 					err "Episode out of range"
 					continue
 				fi
+				[ "$ep_choice_end" = "" ] && break
```

Comment:
I believe it is already checked during input wether ep_chice_end is empty

Reply:
The check likely did exist in a previous version, but in this version:
- `line: 387` sets it
- `line: 392` checks if it is `"-1"`
- `line: 401` converts it to a number and checks if it is less than `$ep_choice_start`

This change is between those lines and breaks out of the `while` before `line: 401` tries to convert `""` to a number